dask/dataframe/core.py

```diff
@@ -1336,6 +1338,17 @@ def elemwise_property(attr, s):
     setattr(Index, name, property(partial(elemwise_property, name)))
 
 
+def nlargest(df, n=5, columns=None):
+    if isinstance(df, Series):
+        token = 'series-nlargest-n={0}'.format(n)
+        f = lambda s: s.nlargest(n)
+    elif isinstance(df, DataFrame):
+        token = 'dataframe-nlargest-n={0}'.format(n)
+        f = lambda df: df.nlargest(n, columns)
+        columns = df.columns  # this is a hack.
+    return aca(df, f, f, columns=columns, token=token)
```

Comment:
Yes. Because ``Index`` is a subclass of ``Series``, it should be a first branch.